distributor-api-v1/operations.md

```diff
@@ -625,8 +625,8 @@ Gives availabilities and pricings for given date interval with product prices in
 | `Id` | string | required | Unique identifier of the rate. |
 | `SettlementType` | string | required | `Automatic`, `Manual`. Determines if system will charge reservation cost automatically or if you'd like employees to manually process payments. |
 | `SettlementAction` | string | required | `ChargeCreditCard`, `CreatePreauthorization`. Determines how payment will be taken at time of automatic trigger. Valid if settlement is automatic only. |
-| `SettlementTrigger` | string | required | Moment that amount is automatically charged, with offset applying to this time (for example, a 'Creation' trigger with no offset will charge the amount when items are created). If settlement is manual, a task will be created at this moment. |
-| `SettlementOffset` | string | required | Time in ISO 8601 format added before or after selected settlement trigger (for example, '-1 day' will charge extent cost 1 day before). |
+| `SettlementTrigger` | string | required | Moment that amount is automatically charged, with offset applying to this time (for example, a 'Creation' trigger with no offset will charge the amount when items are created). If settlement is manual, a task will be created at this moment. Possible triggers: `Confirmation`, `Start`, `End`, `StartDate`, `EndDate`|
```

Comment:
@dobras just one small thing, the sentence should end with a dot here, so it should be:
> Possible triggers: `Confirmation`, `Start`, `End`, `StartDate`, `EndDate`.